### ISSUE TITLE
Belarusian

### ISSUE THREAD

**Issue by mvkarpuk** [squidfunk/mkdocs-material#6043]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "be",
  "direction": "ltr",
  "action.edit": "Правіць старонку",
  "action.skip": "Перайсці да зместа",
  "action.view": "Паглядзець зыходны код старонкі",
  "announce.dismiss": "Больш не паказваць",
  "blog.archive": "Заархіваваць",
  "blog.categories": "Катэгорыі",
  "blog.categories.in": "у",
  "blog.continue": "Працягнуць чытаць",
  "blog.draft": "Чарнавік",
  "blog.index": "Вярнуцца на хатнюю",
  "blog.meta": "Метаданыя",
  "blog.references": "Спасылкі па тэме",
  "clipboard.copy": "Скапіраваць у буфер абмена",
  "clipboard.copied": "Скапіравана ў буфер абмена",
  "consent.accept": "Прыняць",
  "consent.manage": "Кіраваць наладамі",
  "consent.reject": "Адхіліць",
  "footer": "Ніжні калантытул",
  "footer.next": "Наступная",
  "footer.previous": "Папярэдняя",
  "header": "Верхні калантытул",
  "meta.comments": "Каментарыі",
  "meta.source": "Зыходны код",
  "nav": "Навігацыя",
  "readtime.one": "Прачытанне зойме 1 хв",
  "readtime.other": "Прачытанне зойме # хв",
  "rss.created": "RSS стужка",
  "rss.updated": "RSS стужка з абноўленым зместам",
  "search": "Пошук",
  "search.placeholder": "Пошук",
  "search.share": "Падзяліцца",
  "search.reset": "Ачысціць",
  "search.result.initializer": "Пачынаем пошук",
  "search.result.placeholder": "Пачніце друкаваць для пошуку",
  "search.result.none": "Нічога ня знойдзена",
  "search.result.one": "Адзін адпаведны дакумент",
  "search.result.other": "Адпаведных дакументаў: #",
  "search.result.more.one": "Яшчэ 1 на гэтай старонцы",
  "search.result.more.other": "Яшчэ # на гэтай старонцы",
  "search.result.term.missing": "Адсутнічае",
  "select.language": "Выберыце мову",
  "select.version": "Выберыце версію",
  "source": "Перайсці ў рэпазітар",
  "source.file.contributors": "Укладальнікі",
  "source.file.date.created": "Створана",
  "source.file.date.updated": "Апошняе абнаўленне",
  "tabs": "Укладкі",
  "toc": "Змест",
  "top": "Вярнуцца да пачатку"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by mvkarpuk** [comment]
Added Belarusian language.

**Comment by squidfunk** [comment]
Awesome, thanks! Added in 52d32253d.

**Comment by squidfunk** [comment]
Released as part of [9.4.0](https://squidfunk.github.io/mkdocs-material/changelog/#9.4.0).
